src/main/java/frc/team2412/robot/Robot.java

```diff
@@ -139,7 +147,9 @@ public void robotPeriodic() {
 	public void autonomousInit() {
 		Shuffleboard.startRecording();
 		// Basic auto path that travels 1 meter, and then balances on the charge station
-		if (subsystems.drivebaseSubsystem != null) {
+		if (subsystems.drivebaseSubsystem != null && subsystems.armSubsystem != null) {
+			// if manual arm control is enabled in auto, the arm will never move
+			new ManualArmOverrideOffCommand(subsystems.armSubsystem).schedule();
```

Comment:
Please move the null check for the `subsystems.armSubsystem` to only wrap the scheduling of the ManualArmOverrideOffCommand, so that Bonk can still run autonomous paths without an arm connected